pkg/operator/k8sutil/node.go

```diff
@@ -263,10 +263,8 @@ func RookNodesMatchingKubernetesNodes(rookStorage rookalpha.StorageScopeSpec, ku
 			if rookNodeMatchesKubernetesNode(rn, kn) {
 				rn.Name = normalizeHostname(kn)
 
-				// modify the node Location if topology awareness enabled
-				if rookStorage.TopologyAware == true {
-					rn.Location = nodeTopologyLocation(kn, rn.Location)
-				}
+				// modify the node location when running OSD on PVC
```

Comment:
The location could apply to any osd, right? not just on pvcs

Reply:
Yup...